airflow/opt/providers/etna/etna/tests/cassettes/test_metis_files_etl/test_metis_files_etl_e2e.yaml

```diff
@@ -1,4 +1,46 @@
 interactions:
+- request:
+    body: '{"token_type": "task", "project_name": "mvir1", "ready_only": true}'
```

Comment:
random question / nit: is `ready_only` a typo somewhere in the spec that produced this cassette, or perhaps in this YAML file? Should be `read_only`? Maybe that doesn't affect anything, since all task tokens are read_only...

Note that line 45 has the same `ready_only` text.

Reply:
oh snap, good catch.  I think I actually have been fetching write tokens by accident!  I'll make the change to the dictionary.

Reply:
Huh, interesting that it is an `e` token (verified by decoding) -- it may actually be always `e` because of that supereditor thing you fixed last Friday.....it will never be `read_only`, even if you pass in the flag? Perhaps need to rework that logic in `token.rb`?

Reply:
Yeah, I can get a fix in and write some more tests on this branch.

Reply:
@coleshaw Addressed these in latest commit